### PR TITLE
Added stdapi_fs_mount_show, stdapi_net_config_get_proxy and stdapi_sys_eventlog_* functions for Windows in Python meterpreter

### PR DESCRIPTION
**Pull request by ouahibelhanchi**
I have implemented the following functions for Windows:

- stdapi_fs_mount_show
- stdapi_sys_eventlog_open
- stdapi_sys_eventlog_numrecords
- stdapi_sys_eventlog_read
- stdapi_sys_eventlog_clear
- stdapi_sys_eventlog_oldest
- stdapi_sys_eventlog_close
- stdapi_net_config_get_proxy

### CODE REVIEW

**Comment by bcook-r7** [comment]
jenkins, this is OK to test

File: python/meterpreter/ext_server_stdapi.py [hunk 1]
```diff
@@ -238,6 +238,28 @@ class SYSTEM_INFO(ctypes.Structure):
 	class TOKEN_USER(ctypes.Structure):
 		_fields_ = [("User", SID_AND_ATTRIBUTES)]
 
+	class UNIVERSAL_NAME_INFO(ctypes.Structure):
+		_fields_ = [("lpUniversalName", ctypes.c_char_p)]
+
+        class EVENTLOGRECORD(ctypes.Structure):
```
Comment (busterb) [inline python/meterpreter/ext_server_stdapi.py]: there are some whitespace deltas here, will fix

**Comment by busterb** [comment]
```
meterpreter > show_mount

Mounts / Drives
===============

Name  Type    Size (Total)  Size (Free)  Mapped to
----  ----    ------------  -----------  ---------
C:    fixed    63.51 GiB     22.04 GiB
D:    cdrom    56.63 MiB      0.00 B
E:    remote  464.80 GiB    102.85 GiB   \


Total mounts/drives: 3

meterpreter > sysinfo
Computer        : DESKTOP-K2I1LJF
OS              : Windows 10 (Build 15063)
Architecture    : x64
System Language : en_US
Meterpreter     : python/windows
```

**Comment by busterb** [comment]
I wasn't able to get clearev to work as a normal user or administrator. It should work like:
```
meterpreter > clearev
[*] Wiping 307 records from Application...
[*] Wiping 533 records from System...
[*] Wiping 1122 records from Security...
```

But instead:
```
[-] stdapi_sys_eventlog_open: Operation failed: Unknown error
```

**Comment by ouahib-el-hanchi** [comment]
That's weird. I can't seem to replicate the issue.

I have made a test using a normal user and got the following output:

```
meterpreter > sysinfo
Computer        : Lenovo-PC
OS              : Windows 10 (Build 14393)
Architecture    : x64
System Language : en_US
Meterpreter     : python/python
meterpreter > clearev
[*] Wiping 11719 records from Application...
[-] stdapi_sys_eventlog_clear: Operation failed: Windows error: Access is denied.
meterpreter >
```

It is wiping the records from Application event log, but getting an access denied when trying to clear the System log since the user doesn't have the privileges to do so.

When testing using an Administrator user, I get the following output:

```
meterpreter > sysinfo
Computer        : Lenovo-PC
OS              : Windows 10 (Build 14393)
Architecture    : x64
System Language : en_US
Meterpreter     : python/windows
meterpreter > clearev
[*] Wiping 11719 records from Application...
[*] Wiping 0 records from System...
[*] Wiping 0 records from Security...
meterpreter > clearev
[*] Wiping 0 records from Application...
[*] Wiping 0 records from System...
[*] Wiping 0 records from Security...
meterpreter >
```

As you can see, it is clearing the logs correctly.

**Comment by ouahibelhanchi** [comment]
I have implemented the function stdapi_net_config_get_proxy.

```
meterpreter > sysinfo
Computer        : Lenovo-PC
OS              : Windows 10 (Build 14393)
Architecture    : x64
System Language : en_US
Meterpreter     : python/windows
meterpreter > getproxy
Auto-detect     : No
Auto config URL :
Proxy URL       : 127.0.0.1:8081
Proxy Bypass    : <local>
meterpreter >
```

**Comment by busterb** [comment]
Hmm, I wonder if this is a creators-edition issue

**Comment by busterb** [comment]
Confirmed working ok on Windows 7 with Python 2.7.13 with the .msi installer.  Failure happened on Windows 10 Creators Edition using the Python 2.6.1 Web Installer.

I'm going to land this as-is, and make an issue for the later. Thanks!

**Pull request merged**